client/src/modules/Home/Home.module.css

```diff
@@ -173,12 +173,12 @@
     linear-gradient(
       180deg,
       rgba(51, 145, 254, 0.7) 7.5%,
-      rgba(255, 255, 255, 0.7) 79.5%
+      rgba(255, 236, 152, 0.577) 79.5%
```

Comment:
Here we could update to a color variable that includes the opacity for possible consistency and reusability across pages

Reply:
I agree using variables for colors we reuse - tho idk if we will re-use these gradients elsewhere right now - but definitely keeping this in mind in case we do in the future. Like each season background color is different : (